go.mod

```diff
@@ -31,7 +31,7 @@ require (
 	github.com/labstack/gommon v0.4.0
 	github.com/openshift-online/ocm-sdk-go v0.1.373
 	github.com/oracle/oci-go-sdk/v54 v54.0.0
-	github.com/osbuild/images v0.11.0
+	github.com/osbuild/images v0.11.1-0.20231013074322-c3bef149f573
```

Comment:
I'm slightly confused on why this changed to be a specific commit ID now instead of just a tag.

Reply:
Because I update the reference to point to the commit which introduced the support for F40, EL8.10, EL9.4 and s390x/ppc64le on Fedora, instead of doing a release in the `images` repo, which seemed unnecessary (and I still think that it was).

Given the merge conflict and `v0.12.0` being released today, I'll update the PR to use that new version from today as part of rebasing it...

Reply:
✅